### PR TITLE
bug fix: Incomplete timestamp display

### PR DESCRIPTION
**Pull request by geektcp**
bug fix: Incomplete timestamp display
时间戳显示不全，且界面无法设定，且无其他任何地方可以正常显示

### CODE REVIEW

**Comment by wu-sheng** [comment]
Why do you open a new pull request?

**Comment by geektcp** [comment]
> Why do you open a new pull request?

be cause add lang="scss"(ur suggest) , or can u finished it ?

**Comment by wu-sheng** [comment]
The pull request will update after you push codes, there is no need to reopen a new one.

File: src/views/components/trace/trace-chart-table/trace-container.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
         <div class="method">
           Method
         </div>
-        <div class="start-time">
+        <div class="start-time" lang="scss" style="width: 200px;">
```
Comment (Fine0830) [inline src/views/components/trace/trace-chart-table/trace-container.vue]: I mean the style should be here(https://github.com/apache/skywalking-rocketbot-ui/blob/master/src/views/components/trace/trace-chart-table/trace-item.vue#L59)

Comment (geektcp) [reply src/views/components/trace/trace-chart-table/trace-container.vue]: > The pull request will update after you push codes, there is no need to reopen a new one.

ok

Comment (Fine0830) [reply src/views/components/trace/trace-chart-table/trace-container.vue]: You probably update `min-width` to resolve the problem without other change  https://github.com/apache/skywalking-rocketbot-ui/blob/master/src/views/components/trace/trace-chart-table/trace.scss#L26

Comment (geektcp) [reply src/views/components/trace/trace-chart-table/trace-container.vue]: > You probably update `min-width` to resolve the problem without other change https://github.com/apache/skywalking-rocketbot-ui/blob/master/src/views/components/trace/trace-chart-table/trace.scss#L26

Yes,  This code of CSS is complex, Iam not good at CSS,  I think u can fix it

**Pull request closed without merging**